keyboards/id80/iso/keymaps/via/rules.mk

```diff
@@ -0,0 +1,5 @@
+LTO_ENABLE = yes
+VIA_ENABLE = yes
+
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```

Comment:
These shouldn't be needed, as they should already be defined, based on the bootloader type.